vendordeps/PathplannerLib.json

```diff
@@ -1,7 +1,7 @@
 {
     "fileName": "PathplannerLib.json",
     "name": "PathplannerLib",
-    "version": "2024.2.3",
```

Comment:
I'm assuming the library updates are needed too, and you are rolling up many changes into this branch